packages/redux/src/products/selectors/outfits.ts

```diff
@@ -0,0 +1,63 @@
+import { getError, getIsLoading } from '../reducer/outfits';
+import { getProduct } from './product';
+import type { ProductEntity } from '../../entities/types';
+import type { ProductsState } from '../types';
+import type { StoreState } from '../../types';
+
+/**
+ * Returns the loading product outfits condition to a specific product.
```

Comment:
```suggestion
 * Returns the loading status for product outfits of a specific product.
```

Reply:
Fixed. Please review

Reply:
This still seems the same, can you confirm please?

Reply:
Yes is true. Fixed now!
Please confirm